### PR TITLE
Fix kw init wrong template name 

### PR DESCRIPTION
**Pull request by aquilamacedo**
Do not prepend ':' to the template name. Previously, it was required
to set a default template when the `--template` option was not used.
However, this is no longer necessary.

Closes https://github.com/kworkflow/kworkflow/issues/748

### CODE REVIEW

File: documentation/man/features/init.rst [hunk 1]
```diff
@@ -37,6 +37,9 @@ OPTIONS
   Set the variable `default_deploy_target` from **kworkflow.config** to
   *<target>*, which can be local or remote.
 
+-v, \--verbose:
```
Comment (rodrigosiqueira) [inline documentation/man/features/init.rst]: We decided not to support `-v`, please remove it.

File: src/init.sh [hunk 2]
```diff
@@ -47,6 +47,11 @@ function init_kw()
     return 22 # EINVAL
   fi
 
+  if [[ -n "${options_values['VERBOSE']}" ]]; then
+    flag='VERBOSE'
+  fi
+  flag=${flag:-'SILENT'}
```
Comment (rodrigosiqueira) [inline src/init.sh]: How about:
```bash
flag=${flag:-'SILENT'}
[[ -n "${options_values['VERBOSE']}" ]] && flag='VERBOSE'
```

File: src/init.sh [hunk 3]
```diff
@@ -69,17 +74,27 @@ function init_kw()
...
-  cp "$vm_config_file_template" "${PWD}/${KW_DIR}/${vm_name}"
-  cp "$build_config_file_template" "${PWD}/${KW_DIR}/${build_name}"
-  cp "$deploy_config_file_template" "${PWD}/${KW_DIR}/${deploy_name}"
-  cp "$mail_config_file_template" "${PWD}/${KW_DIR}/${mail_name}"
-  cp "$notification_config_file_template" "${PWD}/${KW_DIR}/${notification_name}"
-  cp "$remote_file_template" "${PWD}/${KW_DIR}/${remote_name}"
-
-  sed --in-place --expression "s/USERKW/${USER}/g" -e '/^#?.*/d' "${PWD}/${KW_DIR}/${vm_name}"
-  sed --in-place --expression "s,SOUNDPATH,${KW_SOUND_DIR},g" -e '/^#?.*/d' "${PWD}/${KW_DIR}/${notification_name}"
+  cmd="mkdir --parents \"${PWD}/${KW_DIR}\""
```
Comment (rodrigosiqueira) [inline src/init.sh]: I don't think you need to add `"` inside `"`. In other words, drop the internal quotes for command and the other.

File: documentation/man/features/init.rst [hunk 4]
```diff
@@ -6,8 +6,8 @@ kw-init
 
 SYNOPSIS
 ========
-| *kw* *init* [(-v | \--verbose)] [\--template[=name]]
-| *kw* *init* [(-v | \--verbose)] [\--arch <name>][\--remote <user>@<ip>:<port>][\--target <target>]
```
Comment (rodrigosiqueira) [inline documentation/man/features/init.rst]: You don't need this extra commit for that; go back to the previous commit and just remove `-v` from there.

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]

**Comment by aquilamacedo** [comment]
Hi @rodrigosiqueira. Actually, those changes you asked for were already made in the last commit.

**Comment by rodrigosiqueira** [comment]
@aquilamacedo  squash your last commit with the previous one.

**Comment by rodrigosiqueira** [comment]
Merged!
Thanks a lot

**Pull request closed without merging**